### PR TITLE
Part 2 of ACSC Oct Updates

### PR DESCRIPTION
**Pull request by SamBH**
Part 2 of the updates to the blueprint based on the ACSC's Oct releases. 

Also includes the following additions/fixes:
- Review and update of the Conditional Access policies (design and ABAC)
- Consistency fixes between Office 365 design and ABAC
- Removed reference to 'TrustRecords' in Essential Eight document as not populated if Trusted Documents is not used.

### CODE REVIEW

File: blueprint/platform.md [hunk 1]
```diff
@@ -282,15 +282,16 @@ Conditional Access Policy Design Decisions for all agencies and implementation t
 
 Configuration | Description
 --- | ---
-BLOCK - Legacy Authentication | This global policy blocks all connections from insecure legacy protocols like ActiveSync, IMAP, POP3, etc.
-BLOCK - High-Risk Sign-Ins | This global policy blocks all high-risk authentications (requires Azure AD Premium P2).
 BLOCK - Countries not Allowed | This global policy blocks all connections from countries not in the Allowed countries list.
-GRANT - Terms of Use | This global policy forces Terms of Use on all authentications. Terms of Use is a one-off acceptance, it is used for users to accept their security responsibilities before access is granted. 
+BLOCK - Guess Access | Deny all guest and external users by default.
```
Comment (zedzedzed) [inline blueprint/platform.md:286]: `Guess` should be `Guest`

File: blueprint/office-365.md [hunk 2]
```diff
@@ -122,24 +122,29 @@ The design will take into consideration the services and add-ins that are part o
...
+Bookings              | Disabled        | Exposes a public web page that provides access to user calendars for 3rd parties. There is no requirement to enable the feature as other methods of collaboration are in use. 
+‎Briefing‎ email from ‎Microsoft Viva‎	| Enabled | Enabled by default to improve productivitiy, individual users can unsubscribe if desired.
+Calendar              | Disabled        | External sharing is disabled to prevent potential data spills.
+Cortana               | Disabled        | To align with ACSC Windows 10 hardening guidance to disable Windows web results and Cortana, to limit the risk of accidental disclosure of sensitive information. 
+‎Microsoft‎ communication to users	| Disabled | To prevent communication from Microsoft directly rather than via the Agency's IT team.
+Microsoft Graph Data Connect | Disabled | API connectivity with Azure Data Factory is not currently in scope of the blueprint.
```
[the remark this answers is not in the document]
Comment (SamBH) [reply blueprint/office-365.md:133]: Confirm with Dan that this will not prevent the use of that script.

**Comment by SamBH** [comment]
Updated as per your comments - ready for approval.

**Comment by SamBH** [comment]
Hi Michael,

Have responded to your comments and updated accordingly.
Cheers,

Sam Brazier-Hollins
Managing Consultant
www.oobe.com.au<https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.oobe.com.au%2F&data=04%7C01%7C%7C28e25c8a2f4746eb508b08d937803450%7C05f72b90815c4cb98b4096c29b862189%7C0%7C0%7C637601846075445925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=iAKRFEoJysGq5ANGoKCk5IbhY2VqDN1khNlN9q6fj28%3D&reserved=0>
t: +61 2 6162 1447 | m: +61 409 903 349

***@***.***

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**